presto-main/src/main/java/com/facebook/presto/sql/planner/plan/ExchangeNode.java

```diff
@@ -62,14 +64,17 @@
     // for each source, the list of inputs corresponding to each output
     private final List<List<Symbol>> inputs;
 
+    private final Optional<OrderingScheme> orderingScheme;
```

Comment:
Currently ordering symbols are part of output layout and are materialized. Please add a TODO to not materialize ordering symbols when not needed.